src/components/list/README.md

```diff
@@ -116,14 +116,14 @@ use `<md-divider>` tags.
    <md-list-item *ngFor="#folder of folders">
       <i md-list-avatar class="material-icons">folder</i>
       <h4 md-line>{{folder.name}}</h4>
-      <p md-line class="demo-secondary-text"> {{folder.updated}} </p>
+      <p md-line class="demo-2"> {{folder.updated}} </p>
    </md-list-item>
    <md-divider></md-divider>
    <h3 md-subheader>Notes</h3>
    <md-list-item *ngFor="#note of notes">
       <i md-list-avatar class="material-icons">note</i>
       <h4 md-line>{{note.name}}</h4>
-      <p md-line class="demo-secondary-text"> {{note.updated}} </p>
+      <p md-line class="demo-2"> {{note.updated}} </p>
```

Comment:
That's the secondary text class I'm using in all the other examples.   I must have forgotten to edit this one - changed it to make it consistent.